python/tests/res/enkf/test_enkf_runpath.py

```diff
@@ -58,7 +60,11 @@ def test_with_gen_kw(self):
             self.assertEqual(len(rp), 0)
             rp.load()
             self.assertEqual(len(rp), 1)
+            run_context = None
+            fs = None
+            main = None
```

Comment:
Is this needed even with the tempdir fix? And if it is, should we maybe use `del run_context` rather than assigning `None`, just to be a little more explicit about what is happening? (https://andreasbergstrom.com/posts/python-del-vs-assign-none/) I think this part also may justify a comment, so we don't accidentally remove it in the future.

Reply:
you are right, this `None` is no longer necessary  since `@tmpdir` is in place - so will fix it. As a side note, assigning it to `None` calls via cwrap del automatically.